diff --git a/config.py.sample b/config.py.sample
index 642b753..e23de04 100644
--- a/config.py.sample
+++ b/config.py.sample
@@ -8,8 +8,10 @@ SQLALCHEMY_DATABASE_URI = "postgresql://monuser:monpassachanger@localhost/taxhub
 SQLALCHEMY_TRACK_MODIFICATIONS = False
 
 
-DEBUG=True
+DEBUG = False
 
+TH_HOST = '0.0.0.0'
+TH_PORT = 5000
 
 SESSION_TYPE = 'filesystem'
 SECRET_KEY = 'super secret key'
diff --git a/install_app.sh b/install_app.sh
index 4aa34a8..48364e3 100755
--- a/install_app.sh
+++ b/install_app.sh
@@ -1,13 +1,7 @@
 #!/bin/bash
 
-echo "Arret de l'application..."
-sudo -s supervisorctl stop taxhub
-
 . settings.ini
 
-#Création des répertoires systèmes
-. create_sys_dir.sh
-create_sys_dir
 
 echo "Création du fichier de configuration ..."
 if [ ! -f config.py ]; then
@@ -18,26 +12,6 @@ echo "préparation du fichier config.py..."
 sed -i "s/SQLALCHEMY_DATABASE_URI = .*$/SQLALCHEMY_DATABASE_URI = \"postgresql:\/\/$user_pg:$user_pg_pass@$db_host:$db_port\/$db_name\"/" config.py
 
 
-#installation des librairies
-cd static/
-npm install
-cd ..
-
-#Installation du virtual env
-echo "Installation du virtual env..."
-
-
-if [[ $python_path ]]; then
-  virtualenv -p $python_path $venv_dir
-else
-  virtualenv $venv_dir
-fi
-
-source $venv_dir/bin/activate
-pip install --upgrade pip
-pip install -r requirements.txt
-deactivate
-
 #création d'un fichier de configuration
 if [ ! -f static/app/constants.js ]; then
   echo 'Fichier de configuration non existant'
@@ -47,17 +21,3 @@ fi
 
 #affectation des droits sur le répertoire static/medias
 chmod -R 775 static/medias
-
-#Lancement de l'application
-DIR=$(readlink -e "${0%/*}")
-sudo -s cp taxhub-service.conf /etc/supervisor/conf.d/
-sudo -s sed -i "s%APP_PATH%${DIR}%" /etc/supervisor/conf.d/taxhub-service.conf
-
-
-#création d'un fichier rotation des logs
-sudo cp $DIR/log_rotate /etc/logrotate.d/taxhub
-sudo -s sed -i "s%APP_PATH%${DIR}%" /etc/logrotate.d/taxhub
-sudo logrotate -f /etc/logrotate.conf
-
-sudo -s supervisorctl reread
-sudo -s supervisorctl reload
diff --git a/requirements.txt b/requirements.txt
index c79e588..60daa44 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -10,4 +10,4 @@ Werkzeug==0.11.10
 gunicorn==19.7.0
 Flask-Cors==3.0.3
 git+https://github.com/PnX-SI/UsersHub-authentification-module@1.0.1
-opencv-python==3.3.0.10
+opencv-python>=3.3.0.10
diff --git a/server.py b/server.py
index 55c6adf..e226f70 100644
--- a/server.py
+++ b/server.py
@@ -62,4 +62,4 @@ def init_app():
 app = init_app()
 CORS(app, supports_credentials=True)
 if __name__ == '__main__':
-    app.run()
+    app.run(host='0.0.0.0', port=5000)
